src/directives/compare-to.directive.js

```diff
@@ -0,0 +1,18 @@
+const angular = require('angular');
+
+angular.module('app-bootstrap')
+  .directive('compareTo', function() {
+    return {
+      require: 'ngModel',
+      link: function(scope, element, attrs, ctrl) {
+
+        ctrl.$validators.compareTo = function(val) {
+          return val === scope.$eval(attrs.compareTo);
+        };
```

Comment:
Try to use arrow functions instead of normal 😄 

```suggestion
        ctrl.$validators.compareTo = (val) => val === scope.$eval(attrs.compareTo);
```